schemas/entity-styles.json

```diff
@@ -1,8 +1,55 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Entity Styles",
-  "type": "object",
   "definitions": {
+    "styles": {
```

Comment:
Open to suggestions on better structuring of the schema

How I'd write the TS types:

```ts
type Image = {
  uri: string
  alt: string
}

type Color = {
  color: string
}

type BaseESD = {
  thumbnail: Image
  hero: Image
  background: Color
  text: Color
}

type StyleMode = {
  mode: string
  styles: BaseESD
}

type BasicESD = BaseESD & {
  style_mode: StyleMode
}

type ThemedESD = {
  light: BasicESD
  dark: BasicESD
}

type ESD = BasicESD | ThemedESD
```